packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -15,8 +15,10 @@ const selfUtils = {};
  */
 selfUtils.parse = (self, deviceId) => {
   if (self) {
-    return {
+    const parsedSelf = {
```

Comment:
It doesn't look like this needs assigned to a variable.

Reply:
removed